server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -803,32 +803,23 @@ private TSExecuteStatementResp executeAuthQuery(PhysicalPlan plan) {
     return resp;
   }
 
-  private TSExecuteStatementResp executeDataQuery(PhysicalPlan plan)
+
+  /**
+   * get ResultSet schema
+   */
+  private TSExecuteStatementResp getQueryColumnHeaders(PhysicalPlan plan)
       throws AuthException, TException, QueryProcessException {
     List<Path> paths = plan.getPaths();
     List<String> respColumns = new ArrayList<>();
 
-    // check seriesPath exists
-    if (paths.isEmpty()) {
-      return getTSExecuteStatementResp(getStatus(TSStatusCode.TIMESERIES_NOT_EXIST_ERROR));
-    }
-
-    // check file level set
-    try {
-      checkFileLevelSet(paths);
-    } catch (StorageGroupException e) {
-      logger.error("meet error while checking file level.", e);
-      return getTSExecuteStatementResp(
-          getStatus(TSStatusCode.CHECK_FILE_LEVEL_ERROR, e.getMessage()));
-    }
```

Comment:
Should `CHECK_FILE_LEVEL_ERROR` be removed in `TSStatusCode` and [related User Guide document](https://iotdb.apache.org/#/Documents/progress/chap4/sec2) too? (The error code number is 304)

Reply:
yes, I removed it